docs-site/docs/development/useful_commands.md

```diff
@@ -0,0 +1,30 @@
+---
+sidebar_position: 3
+id: useful_commands
+title: Useful Commands for Development
+---
+
+### Drop, rebuild and seed db
+
+``` bash
+rake db:drop db:create db:migrate db:seed
```

Comment:
**nitpick (blocking):** You can just use `bin/rails` in place if `rake`

Reply:
Fixed.